.circleci/config.yml

```diff
@@ -13,3 +13,15 @@ workflows:
           filters:
             tags:
               only: /^v.*/
+
+      - architect/push-to-docker:
```

Comment:
No context?

Reply:
it somehow worked without it? https://app.circleci.com/pipelines/github/giantswarm/devctl/606/workflows/a58c437f-91bb-4fe0-a9b4-eb59d5f161ec/jobs/636

Reply:
We have [env vars set up for the project](https://app.circleci.com/settings/project/github/giantswarm/devctl/environment-variables?return-to=https%3A%2F%2Fapp.circleci.com%2Fpipelines%2Fgithub%2Fgiantswarm%2Fdevctl%2F606%2Fworkflows%2Fa58c437f-91bb-4fe0-a9b4-eb59d5f161ec%2Fjobs%2F636).

Let's remove them and use context.